test/rebar_test_utils.erl

```diff
@@ -105,12 +105,12 @@ create_config(AppDir, Contents) ->
 
 %% @doc Util to create a random variation of a given name.
 create_random_name(Name) ->
-    random:seed(os:timestamp()),
+    random:seed(erlang:now()),
```

Comment:
This will generate warnings on 18.0+. For tests we might be okay still just using `os:timestamp()`, which won't generate warnings.